swap_meet/vendor.py

```diff
@@ -1,2 +1,67 @@
+from swap_meet import item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory or []
+    
+    def add(self, one_item):
+        if not one_item in self.inventory:
+            self.inventory.append(one_item)
+        return one_item
+    
+    def remove(self, one_item):
+        if not one_item in self.inventory:
+            return False
+        elif one_item in self.inventory:
+            self.inventory.remove(one_item)
+            return one_item
+    
+    def get_by_id(self, id):
+        for item in self.inventory:   
+            if item.id == id:
+                return item
+        return None
+    
```

Comment:
When I was looking at this code in VSCode, it showed me that `item` was never being used. When I looked more into it, I see that's true... and I think we actually don't need a `for` loop in this function at all! We don't need to look at each `item` in `self.inventory`, and we can delete this loop and the tests still pass.